lib/python/docs/notebooks/models_and_releases_demo_pytorch.ipynb

```diff
@@ -141,6 +141,38 @@
     "print(f\"Model card version is {model.model_card_version}.\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Creating and populating a model card\n",
+    "\n",
+    "When creating a model card, first we need to generate an empty one using the `card_from_schema()` method. In this instance, we will use **minimal-general-v10**. You can manage custom schemas using the `Schema()` helper class, but this is out of scope for this demo.\n",
+    "\n",
+    "### Creating and populated a new model card with a template\n",
+    "\n",
+    "When creating a model card from a template, we need to use a preexisting model card as our template. First we create a new model, to create the new model card we use the `card_from_template` method and pass our chosen template model's id. \n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model2 = Model.create(\n",
+    "    client=client, name=\"ResNet-50\", description=\"ResNet-50 model for image classification.\", team_id=\"uncategorised\"\n",
+    ")\n",
+    "\n",
+    "model2_id = model2.model_id\n",
+    "\n",
+    "model2.card_from_template(model2.model_id)\n",
```

Comment:
I think this should use the model id from the previous model?

Reply:
good spot